app/src/main/java/cm/aptoide/pt/home/apps/AppsPresenter.java

```diff
@@ -179,14 +203,34 @@ private void observeUpdatesList() {
         }, error -> crashReport.log(error));
   }
 
+  private void observeAppcUpgradesList() {
+    view.getLifecycleEvent()
+        .filter(lifecycleEvent -> lifecycleEvent == View.LifecycleEvent.CREATE)
+        .observeOn(ioScheduler)
+        .flatMap(__ -> appsManager.getAppcUpgradeDownloadsList())
+        .observeOn(viewScheduler)
+        .doOnNext(list -> view.showAppcUpgradesDownloadList(list))
+        .compose(view.bindUntilEvent(View.LifecycleEvent.DESTROY))
+        .subscribe(created -> {
+        }, error -> crashReport.log(error));
+  }
+
   private void handleResumeUpdateClick() {
     view.getLifecycleEvent()
         .filter(lifecycleEvent -> lifecycleEvent == View.LifecycleEvent.CREATE)
         .observeOn(viewScheduler)
         .flatMap(created -> Observable.merge(view.resumeUpdate(), view.retryUpdate()))
-        .doOnNext(app -> view.setStandbyState(app))
+        .doOnNext(appClickEventWrapper -> {
+          App app = appClickEventWrapper.getApp();
```

Comment:
Since this is being re-used on every method to pause the app i would just create a void method to handle the pausing of the apps and pass the app as an argument of that method